File: 백준/Bronze/10768. 특별한 날/특별한 날.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        String result = "Before";
        int m = Integer.parseInt(br.readLine().trim());
        int d = Integer.parseInt(br.readLine().trim());

        if (m == 2){
            if (d == 18) result = "Special";
            else if (d > 18) result = "After";
        }else if (m > 2){
            result = "After";
        }
        
        bw.write(String.valueOf(result));
        bw.flush();
        br.close();
        bw.close();
    }
}

File: 백준/Bronze/10768. 특별한 날/특별한 날.py
import sys

input = sys.stdin.readline

m = int(input().strip())
d = int(input().strip())
result = "Before"

if m == 2:
    if d == 18:
        result = "Special"
    elif d > 18:
        result = "After"
elif m > 2:
    result = "After"

sys.stdout.write(result + '\n')